beefy-gadget/src/error.rs

```diff
@@ -0,0 +1,9 @@
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
Missing license pre-amble and whole enum docs.

Reply:
Totally forgot about the header